src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/TargetPlatformMonikersValueProvider.cs

```diff
@@ -34,8 +34,10 @@ public override async Task<string> OnGetEvaluatedPropertyValueAsync(string prope
                 ProjectProperties projectProperties = configuredProject.Services.ExportProvider.GetExportedValue<ProjectProperties>();
                 ConfigurationGeneral configuration = await projectProperties.GetConfigurationGeneralPropertiesAsync();
                 string? currentPlatformMoniker = (string?)await configuration.TargetPlatformIdentifier.GetValueAsync();
+                string? currentPlatformVersion = (string?)await configuration.TargetPlatformVersion.GetValueAsync();
+
                 Assumes.NotNull(currentPlatformMoniker);
-                builder.Add(currentPlatformMoniker);
+                builder.Add(currentPlatformMoniker + currentPlatformVersion);
```

Comment:
do you need the Assumes.NotNull() for currentPlatformVersion?

Reply:
Not needed. If there is no version that is fine. We are printing the platform moniker and the version as it is.

Reply:
Got it